barretenberg/ts/src/barretenberg/backend.ts

```diff
@@ -218,19 +216,18 @@ export class UltraHonkBackend {
     const numPublicInputs = Number(proofAsStrings[1]);
```

Comment:
I think this is not right.
You would need to get this from VK like in other places?